Test.C

```diff
@@ -293,12 +293,17 @@ int main(){
         // increase amount in index amount if things are bought
         // decrease credits if things are bought
         char storeSelect;
+
         do{
-            printf("Would you like to buy equipment? y/N: ");
+            printf("Would you like to buy equipment? y/N: \n");
+            printf("Press Y to open the store or ENTER to continue: ");
```

Comment:
You are repeating the same thing twice. I suggest we stick with the first statement and assume they know capital letters mean default, and they can just press enter to achieve that.
```suggestion
            printf("Would you like to buy equipment? y/N: ");
```